assignments/ex2/README.md

```diff
@@ -37,7 +37,7 @@ mode (C) because we use different formats and libraries. (?)
 
 (For implementing _if in the compiler, you can use the virtualized
 `if` that operates on `Rep` types. However, be sure your conditional
-and branches evaluate to `Rep` types, not other types.)
+and branches evaluate to `Rep` types, not other types. Note that the virtualized `if` statements only work in instance methods, and not in lambdas)
```

Comment:
Hmm, I don't understand this note.

Reply:
Inside ImgComp, you can do
```scala
def test(a: Rep[Boolean]) = if (a) true else false
```
But you can't do
```scala
def test() = (a: Rep[Boolean]) => if (a) true else false
```
because the extension of `if` to work on `Rep` only seems to apply within the class methods directly, and not in lambdas.